src/App.js

```diff
@@ -1,43 +1,110 @@
-import React from 'react';
-import { Box, Button, Grid, Container, Divider, Typography } from '@material-ui/core';
+import React, { useState, useEffect } from 'react';
+import {
+	Box,
+	Button,
+	Checkbox,
+	Grid,
+	Container,
+	Divider,
+	Typography,
+	TextField,
+	List,
+	ListItem
+} from '@material-ui/core';
+import DeleteIcon from '@material-ui/icons/Delete';
 import { useSelector, useDispatch } from 'react-redux';
-import { increment, decrement } from './store/features/counter';
+import { add, remove, store } from './store/features/todos';
+import axios from 'axios';
 
 function App() {
-  const dispatch = useDispatch();
-  const state = useSelector(state => state.counter);
-
-  console.log(state);
-
-  return (
-    <Box minHeight="100vh">
-      <Container maxWidth="sm">
-        <Box py={5}>
-          <Typography variant="h1">Todos</Typography>
-          <Divider />
-
-          <Box py={2}>
-            <Typography variant="h3">Counter: {state.value}</Typography>
-          </Box>
-
-          <Box pt={2}>
-            <Grid container spacing={2}>
-              <Grid item>
-                <Button variant="contained" color="primary" onClick={() => dispatch(increment())}>
-                  Add
-                </Button>
-              </Grid>
-              <Grid item>
-                <Button variant="contained" color="primary" onClick={() => dispatch(decrement())}>
-                  Remove
-                </Button>
-              </Grid>
-            </Grid>
-          </Box>
-        </Box>
-      </Container>
-    </Box>
-  );
+	//Request
+	const [ data, setData ] = useState([]);
+
+	useEffect(() => {
+		(async () => {
+			const response = await axios.get('http://167.172.176.146/todos');
```

Comment:
хорошо бы завести глобальную переменную с IP, чтобы в случае изменений не искать по всему коду
```suggestion
// и где-то отдельно этот BASE_URL = 'http://167.172.176.146';
			const response = await axios.get(`${BASE_URL}/todos`);
```